sql_it_test.go

```diff
@@ -361,89 +362,6 @@ func TestSQLWithPortableData(t *testing.T) {
 }
 
 func TestSQLWithPortableDateTime(t *testing.T) {
-	it.SkipIf(t, "hz < 5.0")
```

Comment:
Is this deleted because of missing feature portable on release branch or what? I confused about differences between the PR and other copy.

Reply:
The PR's are supposed to be identical. Let me check...